lib/node_modules/@stdlib/lapack/base/dgttrf/README.md

```diff
@@ -0,0 +1,308 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dgttrf
+
+> Compute an `LU` factorization of a real tri diagonal matrix `A` using elimination with partial pivoting and row interchanges.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var dgttrf = require( '@stdlib/lapack/base/dgttrf' );
+```
+
+#### dgttrf( N, DL, D, DU, DU2, IPIV )
+
+Computes an `LU` factorization of a real tri diagonal matrix `A` using elimination with partial pivoting and row interchanges.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var Int32Array = require( '@stdlib/array/int32' );
+
+var DL = new Float64Array( [ 1.0, 1.0 ] );
+var D = new Float64Array( [ 2.0, 3.0, 1.0 ] );
+var DU = new Float64Array( [ 1.0, 1.0 ] );
+var DU2 = new Float64Array( 1 );
+var IPIV = new Int32Array( 3 );
+
+dgttrf( 3, DL, D, DU, DU2, IPIV );
+// DL => <Float64Array>[ 0.5, 0.4 ]
+// D => <Float64Array>[ 2, 2.5, 0.6 ]
+// DU => <Float64Array>[ 1, 1 ]
+// DU2 => <Float64Array>[ 0 ]
+// IPIV => <Int32Array>[ 0, 1, 2 ]
+```
+
+The function has the following parameters:
+
+-   **N**: order of matrix `A`.
```

Comment:
```suggestion
-   **N**: order of matrix `A`.
```

I suggest updating this description to avoid confusion with "order" as the memory layout. Here, we mean "rank" or the number of rows/columns in a square matrix `A`.